src/main/java/org/fundacion/jala/converter/controller/AudioConverterController.java

```diff
@@ -80,12 +80,12 @@ public String uploadFile(final @RequestParam("file") MultipartFile file,
                 LOGGER.error("Execute Exception" + e.getLocalizedMessage());
             }
         }
-        String outputFilename= ConverterFacade.getAudioConverter(new AudioParameter(storagePath, format, bitrate, hz, volume, audioChannel));
+        String outputFilename = ConverterFacade.getAudioConverter(new AudioParameter(storagePath,
+                format, bitrate, hz, volume, audioChannel));
         String outputPath = FileStorageService.getOutputPath(filename);
         String nameWithoutExtension = outputFilename.substring(0, outputFilename.lastIndexOf(".") + 1);
         extractMetadata(metadata, outputFilename, fileStorageService);
         String pathFile = storagePath.substring(0, storagePath.lastIndexOf(System.getProperty("file.separator")) + 1);
-
         if (!(resultTitle.size() > 0)) {

```

Comment:
Can't it be if (resultTitle.size() <= 0) intead of if (!(resultTitle.size() > 0))  ?

Reply:
changed it.